webapp/src/components/form/FormSelect.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import FormGroup from "./FormGroup";
+import { default as ReactSelect } from "react-select";
+class FormTextBox extends React.Component {
```

Comment:
Shouldn't this be called FormSelect ?

Reply:
Yes, apologies. I think this still worked because we exported by default and then when we imported it FormSelect was used as alias.